play/src/components/CardQuestion.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import {
+  BodyCard,
+  BodyCardContainer,
+} from '../lib/styledcomponents/StyledComponents';
+
+interface CardQuestionProps {
+  questionText: string[];
+  imageUrl: string;
+}
+
+export default function CardQuestion({
```

Comment:
QuestionCard?

Reply:
much better, changed to questioncard and cardanswer has been changed to answercard.